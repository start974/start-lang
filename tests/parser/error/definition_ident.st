def 12 := 3
