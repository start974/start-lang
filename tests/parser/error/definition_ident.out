tests/parser/error/definition_ident.st:1:5
1 | def 12 := 3
        ^^
Error: Expected identifier.
